intel_extension_for_transformers/neural_chat/examples/instruction_tuning/README.md

```diff
@@ -14,12 +14,35 @@ This example demonstrates how to finetune the pretrained large language model (L
 # Prerequisite​
 
 ## 1. Environment​
+### Bare Metal
 Recommend python 3.9 or higher version.
 ```shell
 pip install -r requirements.txt
 # To use ccl as the distributed backend in distributed training on CPU requires to install below requirement.
 python -m pip install oneccl_bind_pt==1.13 -f https://developer.intel.com/ipex-whl-stable-cpu
 ```
+### Docker 
+Pick either one of below options to setup docker environment.
+#### Option 1 : Build Docker image from scratch
+Please refer to this section : [How to build docker images for NeuralChat FineTuning](https://github.com/intel/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/neural_chat/docker/finetuning#4-build-docker-image) to build docker image from scratch.  
+Once you have the docker image ready, please follow [run docker image](https://github.com/intel/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/neural_chat/docker/finetuning#5-create-docker-container) session to launch a docker instance from the image.  
+
+#### Option 2: Pull existed Docker image
+Please follow the session [itrex docker setup](https://github.com/intel/intel-extension-for-transformers/tree/main/docker#set-up-docker-image) and use the docker pull command to pull itrex docker image.  
+Once you have itrex docker image, follow below section to update itrex docker instance for this finetuning example.  
+```shell
+wget https://raw.githubusercontent.com/oneapi-src/oneAPI-samples/master/AI-and-Analytics/Getting-Started-Samples/IntelAIKitContainer_GettingStarted/run_oneapi_docker.sh
+cp requirement.txt /tmp
+# change intel/ai-tools:itrex-0.1.1 according to itrex docker setup session
+./run_oneapi_docker.sh intel/ai-tools:itrex-0.1.1
+# don't need ipex in this sample
+pip uninstall pip install intel_extension_for_pytorch
```

Comment:
If you're just uninstalling IPEX, the command should be:
__pip uninstall pip install intel_extension_for_pytorch__

Reply:
oh. some duplication there. thanks for pointing that out.